packages/analytics-core/src/event-bridge/event-bridge-channel.ts

```diff
@@ -0,0 +1,31 @@
+import { Event, EventBridgeReceiver } from '@amplitude/analytics-types';
+
+export class EventBridgeChannel {
+  channel: string;
+  queue: Event[] = [];
```

Comment:
Wonder if we need a limit in TS as well, just for the normal case and we don't have any receiver, we don't always hold them in memory.

Reply:
@qingzhuozhen Good call, I will fix this. What's the limit in Kotlin?

Reply:
I set the limit to 512, similar to: https://github.com/amplitude/Amplitude-Kotlin/blob/main/event-bridge/src/main/java/com/amplitude/eventbridge/EventBridge.kt#L52